### DISCUSSION TITLE
请问 umi 如何去掉 ant 默认的样式

### DISCUSSION THREAD

**Discussion by boojack** [umijs/umi#5891] [Q&A] [answered]
在官方文档里没看到相关教程。

自带的 ant 样式如下图：
![image](https://user-images.githubusercontent.com/24653555/103215649-f1ec3d00-494e-11eb-87e6-4d56400e0b0f.png)

**Comment by sorrycc** [comment, marked as answer]
为啥要去掉默认样式，是因为没有用到 antd 吗？试试配 antd: false。

**Reply by boojack** [reply]
`antd: false;` 有效。去掉的原因是因为有很多不需要的默认配置，比如 `p {margin-bottom: 1em;}`。
